src/operations/vehicle_bringup/vehicle_bringup/node_vehicle_supervisor.py

```diff
@@ -89,11 +89,19 @@ def send_system_request(self, request_val: bool):
     def timer_callback(self):
         # consolidate current ROS state
         self.ros_state.header.stamp = self.get_clock().now().to_msg()
+
+        # check for timeouts
+        if time.time() - self.planning_update_time > self.timeout:
+            self.ros_state.planning = False
+        if time.time() - self.lidar_update_time > self.timeout:
+            self.ros_state.lidar_operational = False
+        if time.time() - self.sbg_update_time > self.timeout:
+            self.ros_state.sbg_operational = False
+
         if (
-            self.ros_state.sbg_operational
-            # and self.ros_state.lidar_operational
+            self.ros_state.lidar_operational
+            and self.ros_state.sbg_operational
             # and self.ros_state.planning
```

Comment:
Leaving a note here for when nav review branch get's applied. The planner can run on boot so it can be part of G2G checks once again and not run into circular deadlocks again.